clippy_lints/src/loops.rs

```diff
@@ -83,7 +103,8 @@ declare_lint! {
 /// implements `IntoIterator`, so that possibly one value will be iterated,
 /// leading to some hard to find bugs. No one will want to write such code
 /// [except to win an Underhanded Rust
-/// Contest](https://www.reddit.com/r/rust/comments/3hb0wm/underhanded_rust_contest/cu5yuhr).
+/// Contest](https://www.reddit.
+/// com/r/rust/comments/3hb0wm/underhanded_rust_contest/cu5yuhr).
```

Comment:
Are you sure this affected the `clippy_lints` directory as well? In my experience, `cargo fmt` does not touch these by default..